### PR TITLE
[ES] Add `Gameplay/Full combo` and `Gameplay/Combo multiplier effect`; review `Gameplay/Accuracy` and `Gameplay/Choke`

### PR DESCRIPTION
**Pull request by agatemosu**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Gameplay/Choke/es.md [hunk 1]
```diff
@@ -2,15 +2,14 @@
...
 
 ## Causas de los chokes
 
-Una característica clave de un choke es su aleatoriedad y rendimiento inconsistente. Esto excluye errores regulares o específicos del patrón, o se [rompe el deslizante](/wiki/Gameplay/Judgement/Slider_break), que generalmente están vinculados con el nivel de habilidad de un jugador o de cómo se está construido el mapa. Los chockes usualmente son causados por el estrés mental y los nervios excesivos durando el juego, así como por una fatiga general.
+Una característica clave de un choke es su aleatoriedad y rendimiento inconsistente. Esto excluye fallos regulares o específicos de un patrón o la [rotura de un slider](/wiki/Gameplay/Judgement/Slider_break), que generalmente están vinculados con el nivel de habilidad de un jugador o de cómo se está construido el beatmap. Los chockes usualmente son causados por el estrés mental y los nervios excesivos durando el juego, así como por una fatiga general.
```
Comment (UnknownOsu) [inline wiki/Gameplay/Choke/es.md:15]:
```suggestion
Una característica clave de un choke es su aleatoriedad y rendimiento inconsistente. Esto excluye fallos regulares o específicos de un patrón o la [rotura de un slider](/wiki/Gameplay/Judgement/Slider_break), que generalmente están vinculados con el nivel de habilidad de un jugador o de cómo se está construido el beatmap. Los chokes usualmente son causados por el estrés mental y los nervios excesivos durando el juego, así como por una fatiga general.
```

File: wiki/Gameplay/Full_combo/es.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+---
+stub: true
+tags:
+  - FC
+  - full combo
+---
+
+# Combo completo
+
+**Full combo** o **combo completo** (o *FC* para acortar) es un término usado para describir el [combo](/wiki/Beatmapping/Combo) máximo que puede conseguir un jugador en un [beatmap](/wiki/Beatmap). Los combos completos se puntúan pasando un beatmap sin fallos, [sliders rotos](/wiki/Gameplay/Judgement/Slider_break) o [slider ends](/wiki/Gameplay/Hit_object/Slider/Slidertail) fallados.
```
Comment (UnknownOsu) [inline wiki/Gameplay/Full_combo/es.md:10]:
```suggestion
**Full combo** o **combo completo** (o *FC* para acortar) es un término usado para describir el [combo](/wiki/Beatmapping/Combo) máximo que puede conseguir un jugador en un [beatmap](/wiki/Beatmap). Los combos completos se consiguen pasando un beatmap sin fallos, [slidersbreaks](/wiki/Gameplay/Judgement/Slider_break) o [slider ends](/wiki/Gameplay/Hit_object/Slider/Slidertail) fallados.
```

File: wiki/Gameplay/es.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+- [Hyperdash](/wiki/Gameplay/Hyperdash)
+- [Input device](/wiki/Gameplay/Input_device)
+- [Juicio](/wiki/Gameplay/Judgement)
+- [Kiai time](/wiki/Gameplay/Kiai_time)
+- [Mind block](/wiki/Gameplay/Mind_block)
+- [Notelock](/wiki/Gameplay/Judgement/Notelock)
+- [Pass](/wiki/Gameplay/Pass)
+- [Tiempo de juego](/wiki/Beatmap/Play_time)
+- [Repetición](/wiki/Gameplay/Replay)
+- [Puntación](/wiki/Gameplay/Score)
```
Comment (UnknownOsu) [inline wiki/Gameplay/es.md:25]:
```suggestion
- [Puntuación](/wiki/Gameplay/Score)
```

File: wiki/Gameplay/Full_combo/es.md (hunk 2 above)
Comment (UnknownOsu) [reply wiki/Gameplay/Full_combo/es.md:10]: Está raro `slider rotos`. Además, `slider ends` tampoco está traducido, así que creo que es mejor que también se quede en su forma original (`sliderbreaks`)

**Review by UnknownOsu — CHANGES_REQUESTED** [review]

File: wiki/Gameplay/Choke/es.md [hunk 4]
```diff
@@ -2,15 +2,14 @@
...
 
 ## Causas de los chokes
 
-Una característica clave de un choke es su aleatoriedad y rendimiento inconsistente. Esto excluye errores regulares o específicos del patrón, o se [rompe el deslizante](/wiki/Gameplay/Judgement/Slider_break), que generalmente están vinculados con el nivel de habilidad de un jugador o de cómo se está construido el mapa. Los chockes usualmente son causados por el estrés mental y los nervios excesivos durando el juego, así como por una fatiga general.
+Una característica clave de un choke es su aleatoriedad y rendimiento inconsistente. Esto excluye fallos regulares o específicos de un patrón o la [rotura de un slider](/wiki/Gameplay/Judgement/Slider_break), que generalmente están vinculados con el nivel de habilidad de un jugador o de cómo se está construido el beatmap. Los chokes usualmente son causados por el estrés mental y los nervios excesivos durando el juego, así como por una fatiga general.
```
Comment (UnknownOsu) [inline wiki/Gameplay/Choke/es.md:15]:
```suggestion
Una característica clave de un choke es su aleatoriedad y rendimiento inconsistente. Esto excluye fallos regulares o específicos de un patrón o [sliderbreaks](/wiki/Gameplay/Judgement/Slider_break), que generalmente están vinculados con el nivel de habilidad de un jugador o de cómo está construido el beatmap. Los chokes usualmente son causados por el estrés mental y los nervios excesivos durante el juego, así como por una fatiga general.
```

File: wiki/Gameplay/Full_combo/es.md [hunk 5]
```diff
@@ -0,0 +1,16 @@
+---
+stub: true
+tags:
+  - FC
+  - full combo
+---
+
+# Combo completo
+
+**Full combo** o **combo completo** (o *FC* para acortar) es un término usado para describir el [combo](/wiki/Beatmapping/Combo) máximo que puede conseguir un jugador en un [beatmap](/wiki/Beatmap). Los combos completos se consiguen pasando un beatmap sin fallos, [slidersbreaks](/wiki/Gameplay/Judgement/Slider_break) o [slider ends](/wiki/Gameplay/Hit_object/Slider/Slidertail) fallados.
```
Comment (UnknownOsu) [inline wiki/Gameplay/Full_combo/es.md:10]:
```suggestion
**Full combo** o **combo completo** (o *FC* para acortar) es un término usado para describir el [combo](/wiki/Beatmapping/Combo) máximo que puede conseguir un jugador en un [beatmap](/wiki/Beatmap). Los combos completos se consiguen pasando un beatmap sin fallos, [sliderbreaks](/wiki/Gameplay/Judgement/Slider_break) o [slider ends](/wiki/Gameplay/Hit_object/Slider/Slidertail) fallados.
```

**Review by UnknownOsu — CHANGES_REQUESTED** [review]

**Review by TicClick — APPROVED** [review]

**Pull request merged**